snmp/__init__.py

```diff
@@ -0,0 +1,315 @@
+#!/usr/bin/env python3
+# vim: set encoding=utf-8 tabstop=4 softtabstop=4 shiftwidth=4 expandtab
+#########################################################################
+#  Copyright 2019-2020 Michael Wenzel               wenzel_michael@web.de
+#########################################################################
+#  This file is part of SmartHomeNG.    https://github.com/smarthomeNG//
+#
+#  SmartHomeNG is free software: you can redistribute it and/or modify
+#  it under the terms of the GNU General Public License as published by
+#  the Free Software Foundation, either version 3 of the License, or
+#  (at your option) any later version.
+#
+#  SmartHomeNG is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+#  GNU General Public License for more details.
+#
+#  You should have received a copy of the GNU General Public License
+#  along with SmartHomeNG. If not, see <http://www.gnu.org/licenses/>.
+#########################################################################
+
+from lib.module import Modules
+from lib.model.smartplugin import *
+from lib.item import Items
+
+import threading
+import logging
+import time
+import struct
+from puresnmp import get
```

Comment:
here u are using puresnmp, in requirements.txt you have pysnmp! Hence, I am getting an error:

Plugin 'nas' error importing Python package: No module named 'puresnmp'

Reply:
corrected in requirments.txt